packages/web3-eth-abi/src/utils.ts

```diff
@@ -179,7 +179,7 @@ export const formatParam = (type: string, _param: unknown): unknown => {
 	// Format correct width for u?int[0-9]*
 	let match = paramTypeNumber.exec(type);
 	if (match) {
-		const size = parseInt(match[2] ?? '256', 10);
+		const size = parseInt(match[2] ? match[2] : '256', 10);
```

Comment:
if i pass `int` match[2] will be empty string `''`. `??` this operator only works with nullish results